ci/Jenkinsfile.gh-pages

```diff
@@ -28,6 +28,16 @@ pipeline {
       }
     }
 
+    stage('Examples') {
```

Comment:
Not sure if you have done it but you first need to run, once, from root:

```
npm ci
npm run build
```

And then go in each example folders to run (can be done in parallel)

```
npm ci
npm run build
````

Reply:
That's what I'm doing, just with yarn, look at the function.

Reply:
I think you are missing a `npm run build` at the root. If I understand correctly, you are only running `yarn run doc` at the root.

I tried to test myself: https://github.com/status-im/js-waku/pull/437 but I could not find how to run a jenkins with my changes.

Reply:
Thank you but I don't understand why https://github.com/status-im/js-waku/pull/437 does not have any Jenkins build in the dashboard